orion/pkg/domains/hash_test.go

```diff
@@ -0,0 +1,33 @@
+package domains
+
+import (
+	"testing"
+)
+
+func TestHash(t *testing.T) {
+	var hash Hash
+	err := hash.GetHash("potato")
```

Comment:
hmm I think the usage would make more sense if we had something like this:
`hash, err := CreateHash("potato")`

It would be great if I had a utility function that actually created a hashed password from a string.

Reply:
> It would be great if I had a utility function that actually created a hashed password from a string.
So like generate a password for the user to use as their actual password?